.github/workflows/build-release.yaml

```diff
@@ -42,8 +42,46 @@ jobs:
           docker tag defichain-x86_64-pc-linux-gnu:${ver} defi/defichain:${tag}
         done
         docker push defi/defichain:${ver}
+    
+  linux-armhf:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Populate environment
+      run: GIT_VERSION=1 ./make.sh ci-export-vars
+
+    - name: Build and package
+      run: GIT_VERSION=1 TARGET="arm-linux-gnueabihf" ./make.sh docker-release
 
-  windows:
+    - name: Publish artifact - arm-linux-gnueabihf
```

Comment:
Add docker image publishing as well for the given platform.

Reply:
Resolved in commit 0220fb97d. Docker images tag convention is now defi/defichain:{ver}-linux-x64 and defi/defichain:{ver}-linux-arm.